tfs/src/main/java/hudson/plugins/tfs/rm/ReleaseManagementCI.java

```diff
@@ -163,6 +232,8 @@ void CreateRelease(
             JSONObject object = new JSONObject(response);
```

Comment:
are we not converting to ReleaseDetails? if so then ReleaseDetails is not used anymore. If its not used anymore please remove the ReleaseDetails type.